lib/rubocop/cop/minitest/refute_false.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Check if your test uses `refute(actual)`
```

Comment:
The wording here is confusing. I'd write it the other way around or "This cop enforces the usages of `refute` over `assert...`. 

I'd also use `object` instead of `actual` here give there's nothing to compare against.